### PR TITLE
Added FDA intensity limits

### PR DESCRIPTION
**Pull request by mofii**

### CODE REVIEW

File: src/metrics.jl [hunk 1]
```diff
@@ -108,3 +108,65 @@ function ptp_pressure(pressures::Array{T, N}) where {T <: Unitful.Pressure, N}
...
+    sppa_limit = Dict(
+            "peripheal_vessel" => 190u"W/cm^2",
+            "cardiac"          => 190u"W/cm^2",
+            "fetal"            => 190u"W/cm^2",
+            "abdominal"        => 190u"W/cm^2",
+            "intraoperative"   => 190u"W/cm^2",
+            "pediatric"        => 190u"W/cm^2",
+            "small_organ"      => 190u"W/cm^2",
+            "cathalic"         => 190u"W/cm^2",
+            "ophthalmic"       => 28u"W/cm^2",
```
Comment (stevenaleung) [inline src/metrics.jl:153]: Looks like there was an extra tab here!

File: src/metrics.jl [hunk 2]
```diff
@@ -108,3 +108,65 @@ function ptp_pressure(pressures::Array{T, N}) where {T <: Unitful.Pressure, N}
     ptp_pressure = pp_pressure(pressures) .+ pn_pressure(pressures)
     return ptp_pressure
 end
+
+
+"""
+get_fda_limits(body_part::String)
+Valid input string values are:
+    "peripheal_vessel", "cardiac", "fetal", "abdominal", "intraoperative",
```
Comment (stevenaleung) [inline src/metrics.jl:116]: `peripheral` is misspelled at multiple locations

Comment (stevenaleung) [inline src/metrics.jl:116]: Is `intraoperative` needed?

File: src/metrics.jl [hunk 3]
```diff
@@ -108,3 +108,65 @@ function ptp_pressure(pressures::Array{T, N}) where {T <: Unitful.Pressure, N}
...
+    "pediatric", "small_organ", "cathalic", "ophthalmic".
```
Comment (stevenaleung) [inline src/metrics.jl:117]: `cephalic` is misspelled as well

**Review by stevenaleung — APPROVED** [review]
Good to go after a couple of spelling corrections!

File: src/metrics.jl (hunk 3 above)
Comment (mofii) [reply src/metrics.jl:117]: nice catch

File: src/metrics.jl (hunk 2 above)
Comment (mofii) [reply src/metrics.jl:116]: Not currently for us. But it's one of the listed groups and it might be useful for others.

**Pull request merged**